kategory-docs/docs/docs/datatypes/option/README.md

```diff
@@ -19,13 +19,25 @@ Kategory models the absence of values through the `Option` datatype similar to h
 import kategory.*
 import kategory.Option.*
 
-val someValue: Option<String> = Some("I am wrapped in something")
+val message: String? = "I am wrapped in something"
+
+val someValue: Option<String> = Some(message)
 someValue
+
+// Or with the toOption extention
```

Comment:
Only the last return of a block is evaluated and returned by ank. You need to split these examples into two ```kotlin:ank blocks, one that returns someValue and another for otherValue